apps/frontend/src/views/Login.vue

```diff
@@ -99,7 +112,30 @@ export default class Login extends Vue {
     this.checkLoggedIn();
   }
 
+  getCookie(name: string) {
+    function escapeName(s: string) {
+      return s.replace(/([.*+?\^$(){}|\[\]\/\\])/g, '\\$1');
+    }
+    const match = document.cookie.match(
+      RegExp(`(?:^|;\\s*)${escapeName(name)}=([^;]*)`)
+    );
+    return match ? match[1] : null;
+  }
+
+  delete_cookies(names: string[]) {
+    names.forEach((name) => {
+      document.cookie =
+        name + '=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;';
+    });
+  }
```

Comment:
Would the vue cookies library work in place of this?

https://www.npmjs.com/package/vue-cookies